Makefile

```diff
@@ -1,18 +1,41 @@
-.PHONY: dev clean e2e push
+.PHONY: clean dev dev-private-cluster update-image-on-deployment push-tester-image deploy-e2e run-e2e
+
+# keep separate for simultaneous public/private dev without need for resource recreation
+clean-public:
```

Comment:
would be good to just have a `clean` that does both as well

Reply:
makes sense!